backend/src/Request/Captain/CaptainProfileResponse.php

```diff
@@ -0,0 +1,42 @@
+<?php
+
+namespace App\Response\Captain;
+
+class CaptainProfileResponse
+{
+    public int $captainId;
+
+    public null|string $captainName;
```

Comment:
I have suggestion to use annotation instead of this way of type hint (I mean when we have more than one type for the variable).